Explain boolean conversion to numeric type. (#13304)
dotnet/docs: docs/visual-basic/language-reference/operators/orelse-operator.md

|`False`|`False`|`False`|  
  
## Data Types  

>  The `OrElse` operator is defined only for the [Boolean Data Type](../../../visual-basic/language-reference/data-types/boolean-data-type.md). Visual Basic converts each operand as necessary to `Boolean` and performs the operation entirely in `Boolean`. If you assign the result to a numeric type, Visual Basic converts it from `Boolean` to that type such that `False` becomes `0` and `True` becomes `-1`.

Reviewer: ```suggestion
 The `OrElse` operator is defined only for the [Boolean Data Type](../../../visual-basic/language-reference/data-types/boolean-data-type.md). Visual Basic converts each operand as necessary to `Boolean` before evaluating the expression. If you assign the result to a numeric type, Visual Basic converts it from `Boolean` to that type such that `False` becomes `0` and `True` becomes `-1`.
```

>  [See Boolean Type Conversions](../../../visual-basic/language-reference/data-types/boolean-data-type#type-conversions)

Reviewer: ```suggestion
For more information, see [Boolean Type Conversions](../data-types/boolean-data-type.md#type-conversions)
```
